fca_import_scripts/lib/ext_to_sql.rb

```diff
@@ -0,0 +1,133 @@
+require 'csv'
+require 'date'
+require 'yaml'
+
+class ExtToSql
+  TIMESTAMP = DateTime.now.strftime('%Y-%m-%d %H:%M:%S.%N')
+  REPAIR_FILE = File.join(File.dirname(__FILE__), '..', 'repairs.yml')
+
+  def initialize(stderr = nil)
+    @stderr = stderr
+  end
+
+  def process_ext_file(path, &block)
+    File.foreach(path) do |line|
+      line = repair_line line
+      row = line.split('|')
+
+      next if row[0] == 'Footer'
+
+      if row[0] == 'Header'
+        @type = determine_type_from_header(row)
```

Comment:
actually no. it's still defined in that loop. whatever — it's not ideal, now we've got a class I'm going to make it a real instance variable.